### PR TITLE
fix(DB/smart_scripts): The Plains Vision speed

### LINKED ISSUE #5982: Rite of Vision quest's ghost wolf running too fast
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [ ] Alliance
- [x] Horde

##### CONTENT PHASE:

- [ ] Generic
- [x] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

Original report: https://github.com/chromiecraft/chromiecraft/issues/680

##### CURRENT BEHAVIOUR:

While doing quest "Rite of Vision" ( https://wowgaming.altervista.org/aowow/?quest=772 ), we are asked to follow a ghost wolf to a secret cave. The issue is that the ghost wolf runs too fast and it makes it very hard to follow the wolf if you do not already know where it is running to. In Classic WoW the wolf runs more calmly (as it is a low level quest so players might no be very used to the game yet).
Here is a video of the run on ChromieCraft, you can see how fast the ghost goes.
https://www.youtube.com/watch?v=IBlNTS2i9p4

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Ghost wolf should run slower and let the player catch up.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Take quest Rite of Vision (requires to complete other quests).
2. Take the drug (just in game silly).
3. See how fast the ghost wolf is running away.

##### EXTRA NOTES:

N/A.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/e50754d804987ec93792b6a4d113913ab018189b

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

Confirmed and reproduced on a local AC server (AC HASH/COMMIT: a9e3d8d):

WoWScrnShot_052121_101845
WoWScrnShot_052121_101926
WoWScrnShot_052121_101952
WoWScrnShot_052121_102005

I would add saying that not only does the wolf run more than necessary, but at times the animation fails and it glides levitating a foot off the ground ^^ 

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/98598885-rite-of-vision-quest-s-ghost-wolf-running-too-fast?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by anguaive**
During the Rite of Vision quest (id: 772), The Plains Vision creature
(entry: 2983) guides the player to a location. The creature moves too
fast and outruns the player, making it impossible to follow all the way.

## Changes Proposed:
- Adjust The Plains Vision movement from running to walking

## Issues Addressed:

- Closes #5982
- Closes https://github.com/chromiecraft/chromiecraft/issues/680

## SOURCE:

https://www.youtube.com/watch?v=BYy_-uytlrg

## Tests Performed:

- Tested in-game

## How to Test the Changes:

1. `.quest add 772`
2. `.go o 20436`
3. Use quest item and follow the ghost wolf

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Eldarix** [comment]
Checked, now you can follow the little spirit wolf without problem, you even have to wait for him, the speed seems something like 0.75, I'm not sure if it is the intention:

WoWScrnShot_060421_064946
WoWScrnShot_060421_065023
WoWScrnShot_060421_065316

**Comment by anguaive** [comment]
Yup that's intended. It also moves slowly in the classic video I've linked (starts around 3:40)

**Review by r-o-b-o-t-o — APPROVED** [review]
